### PR TITLE
[Ready] Makes Cargo Toy Packet "TODO" a dream come true

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
:cl: 
tweak: cargo toy crate now has more toys
/:cl: 

[Changelogs]
// TODO make this actually just use the arcade machine loot list
More like!
// TODID make this actually just use the arcade machine loot list 
Yay!
Also sets a reminder to new toys being added to the arcade machine to also place them into the cargo pack!
[why]
It says TODO, and I already had it done for, so mite as well send it upstream a bit

### CODE REVIEW

**Comment by cyclowns** [comment]
i think the TODO was asking for a shared list of toys to be made between the crate/arcade machines so that you didn't have to update each individually, not just copying over the list to the crate

**Comment by Tlaltecuhtli** [comment]
can you increase the amount to 10? its already a meme crate

**Comment by anconfuzedrock** [comment]
but then we won't be able to get plushiiiiiiiiiieeeeeeeees!

**Comment by Trilbyspaceclone** [comment]
@anconfuzedrock Fix'ed!

**Comment by kingoftheautism** [comment]
meme PR, though I would change it to 5 random arcade prize items, instead of putting them all together in one crate in amounts of 10. though it's meme PR, so why not

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -1897,33 +1897,68 @@
...
-	                /obj/item/toy/talking/griffin,
-	                /obj/item/toy/nuke,
-	                /obj/item/toy/minimeteor,
-	                /obj/item/toy/plush/carpplushie,
+ /datum/supply_pack/costumes_toys/randomised/toys
+ 	name = "Toy Crate"
+ 	desc = "Who cares about pride and accomplishment? Skip the gaming and get straight to the sweet rewards with this product! Contains five random toys. Warranty void if used to prank research directors."
+ 	cost = 5000 // or play the arcade machines ya lazy bum
+ 	// TODID make this actually just use the arcade machine loot list
+ 	num_contained = 10
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs.dm]: Ten Items or Five, VOTE
👍 = 10 Items
🎉 = 5 Items

**Comment by ExcessiveUseOfCobblestone** [comment]
WHY DO ARCADE MACHINES HAVE THEIR OWN STATIC-BUT-NOT LIST!!!

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (kingoftheautism) [reply code/modules/cargo/packs.dm]: actually, each item in amount of 10, or just 10 random items in amount of 1?

Comment (ExcessiveUseOfCobblestone) [reply code/modules/cargo/packs.dm]: the latter, don't scare me.

Comment (Trilbyspaceclone) [reply code/modules/cargo/packs.dm]: Closing the vote on Fri Nov 9th

**Comment by k-k-karma** [comment]
Verified WebEditor

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (Bawhoppen) [reply code/modules/cargo/packs.dm]: Please don't vote for 10. We don't need to have every toy in every crate every time.

**Comment by ExcessiveUseOfCobblestone** [comment]
ping me on coderbus discord or coding-general (moo1969 with @ prefix)

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -1959,30 +1959,20 @@
...
 	crate_name = "toy crate"
 	crate_type = /obj/structure/closet/crate/wooden
 
+/datum/supply_pack/costumes_toys/randomised/toys/generate()
+	. = ..()
+	for(var/i in 1 to num_contained)
+		var/the_toy = pickweight(GLOB.arcade_prize_pool)
+		new the_toy(.)
+	else
+		the_toy = pickweight(list(subtypesof(/obj/item/toy/plush)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/cargo/packs.dm]:
```suggestion
		the_toy = pickweight(list(subtypesof(/obj/item/toy/plush))
```

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -1959,30 +1959,20 @@
...
-	                /obj/item/toy/redbutton,
-					/obj/item/toy/eightball,
-					/obj/item/vending_refill/donksoft)
+	contains = list()
 	crate_name = "toy crate"
 	crate_type = /obj/structure/closet/crate/wooden
 
+/datum/supply_pack/costumes_toys/randomised/toys/generate()
+	. = ..()
+	for(var/i in 1 to num_contained)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/cargo/packs.dm]:
```suggestion
	for(var/i in 1 to num_contained)
	    if(prob(50)) //maybe?
```

File: code/modules/cargo/packs.dm [hunk 4]
```diff
@@ -1959,30 +1959,20 @@
...
+		var/the_toy = pickweight(GLOB.arcade_prize_pool)
+		new the_toy(.)
+	else
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/cargo/packs.dm]:
```suggestion
	    else
```

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
needs if as well as shift in else

**Comment by Trilbyspaceclone** [comment]
Yay

**Pull request merged**